string-manipulation-2/src/last-chars.ts

```diff
@@ -0,0 +1,12 @@
+/* exported lastChars */
+function lastChars(length: number, str: string): string {
+  let newStr = '';
+  if (str.length <= length) {
+    return str;
+  } else {
+    for (let i = str.length - length; i < str.length; i++) {
```

Comment:
initial:
let i = Math.max(0, str.length - length)
will ensure we never start at a negative index